ParticleSpy/ParticleAnalysis.py

```diff
@@ -285,7 +285,7 @@ class parameters(object):
     """A parameters object."""
     
     def generate(self,threshold='otsu',watershed=False,watershed_size=50,
-                 watershed_erosion=5,invert=False,min_size=0,store_im=False,
+                 watershed_erosion=5,invert=False,min_size=0,store_im=True,
```

Comment:
I'm not sure setting the default value to True is a good idea as some people might run in to memory issues. I'd prefer to keep it as False.